src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -2454,7 +2454,7 @@ void M()
     {
         $$";
 
-            await VerifyItemExistsAsync(markup, "M", expectedDescriptionOrNull: $"void C.M(int i) (+ 1 {FeaturesResources.Overload})");
+            await VerifyItemExistsAsync(markup, "M", expectedDescriptionOrNull: $"void C.M(int i) (+ 1 {FeaturesResources.Overload})");
```

Comment:
If I take the raw file contents of this file from Master and paste them over the copy in my enlistment, Git shows no diff between what I committed and what I pasted, which makes me think this is a GitHub bug. Unless there's a clipboard issue?